modules/distribution-workflowoperation/src/main/java/org/opencastproject/workflow/handler/distribution/PublishEngageWorkflowOperationHandler.java

```diff
@@ -734,10 +730,4 @@ private void retractFromEngage(MediaPackage mediaPackage) throws WorkflowOperati
       logger.error("Retraction failed of Mediapackage: { }", mediaPackage.getIdentifier().toString(), ex);
     }
   }
-
-  /** OSGi DI */
-  protected void setSecurityService(SecurityService securityService) {
```

Comment:
Removing this causes

```
2019-05-28 13:46:27,381 | ERROR | (PaxLoggingServiceImpl$1ManagedPaxLoggingService:398) - bundle opencast-distribution-workflowoperation:8.0.0.SNAPSHOT (78)[org.opencastproject.workflow.handler.distribution.PublishAWSWorkflowOperationHandler(62)] : bind method [setSecurityService] not found; Component will fail
2019-05-28 13:46:27,382 | ERROR | (PaxLoggingServiceImpl$1ManagedPaxLoggingService:398) - bundle opencast-distribution-workflowoperation:8.0.0.SNAPSHOT (78)[org.opencastproject.workflow.handler.distribution.PublishAWSWorkflowOperationHandler(62)] : bind method [setSecurityService] not found
```

Reply:
I fixed the reference myself since the rest of this looks right.

Reply:
Thanks!